hologram_dashboard/fetcher/hologram_availability_fetcher.h

```diff
@@ -0,0 +1,78 @@
+/*
+    Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+#pragma once
+#include <iostream>
+#include <string>
+#include <unordered_map>
+#include <gflags/gflags.h>
+#include <assert.h>
+#include "fetcher/proto/hologram_availability.pb.h"
+#include "fetcher/proto/hologram_config.pb.h"
+
+DEFINE_string(chipper_batch_job_cell, "undefined", "Specifies the server on which chipper file location is stored");
+DEFINE_string(chipper_GDPR_batch_job_cell, "undefined", "Specifies the server on which chipperGDPR file location is stored");
+DEFINE_string(prime_meridian_batch_job_cell, "undefined", "Specifies the server on which prime meridian file location is stored");
+DEFINE_string(config_file_path, "undefined", "Specifies where the config file can be located");
+
+namespace wireless_android_play_analytics {
+    class HologramAvailabilityFetcher {
```

Comment:
No indent for class HologramAvailabilityFetcher.

namespace wireless_android_play_analytics {
class HologramAvailabilityFetcher {


}

} // namespace wireless_android_play_analytics

Reply:
done